src/pages/index.astro

```diff
@@ -354,13 +354,15 @@ const sides_id = ['left', 'right'];
 					switch (result.message) {
 						case 'Member Exists':
 						case 'email subscribed':
+							button.classList.remove('bg-gray-light hover:bg-gray-dark');
 							button.innerHTML = 'Subscribed';
 							button.style.backgroundColor = '#0fa';
```

Comment:
There is still a raw color code in the code.
Maybe make it as class?

Reply:
yep I know, this is why it's part 1 haha
I need components that are in dev to finish this (I need to move some stuff in the newly created utils folder)